app/views/activities/_form.html.slim

```diff
@@ -36,12 +46,23 @@
           .col-6
             = f.input :duration_journey
 
+
   .card.mt-3
     .card-body
       .form-group
         label{for='activity_activity_documents'} = t('activerecord.attributes.activity.activity_documents')
-        br
-        = f.file_field :activity_documents, multiple: true
+        .row
+          .col-6
+            = f.file_field :activity_documents, multiple: true
+          .col-6
+            ul.list-group
+              - @activity.activity_documents.each do |document|
```

Comment:
hmm...like this we use the activity controllers destroy method to destroy an attachement. what i usually do is, to use `  accepts_nested_attributes_for :activity_documents`, allow_destroy: true` which allows you to use an inner form:

I'm sure this won't work straight away, but 

```rb
  - if @activity.activity_documents.any?
      = form.simple_fields_for :activity_documents do |activity_document|
          - document = activity_document.object
          = activity_document.input :id, as: :hidden, input_html: { value: pic.id }, wrapper: false
          p = document.filename
          = activity_document.input :_destroy, as: :boolean, label: t('buttons.destroy')
```

in the controller you have to permit those attributes as well:

```rb
    params.require(:activity).permit(:label, :description, :language, :js_type, :participants_count_activity,
                                     :participants_count_transport, :duration_activity, :duration_journey, :location,
                                     I18n.available_locales.map { |l| :"label_#{l.to_s}"},
                                     I18n.available_locales.map { |l| :"description_#{l.to_s}"},
                                     I18n.available_locales.map { |l| :"label_#{l}" },
                                     I18n.available_locales.map { |l| :"description_#{l}" },
                                     stufe_ids: [], stufe_recommended_ids: [], goal_ids: [], tag_ids: [],
                                     activity_documents: [], # this might be still needed to add new files..not sure here! but if  you are intrested how to do this also with inner forms, i once did the trick!
)
{ activity_documents_attributes: %i[id filename _destroy] }
```